sklearn/decomposition/_lda.py

```diff
@@ -496,13 +497,20 @@ def partial_fit(self, X, y=None):
         self
         """
         self._check_params()
-        X = self._check_non_neg_array(X,
+        first_time = not hasattr(self, 'components_')
+
+        # deactivating check for now (specific tests about error message would
```

Comment:
As this case only happens in 2 places in the whole code-base, I'd rather make the comment clearer instead of updating the SLEP (I'm not even sure we can do that?)

As soon as this PR is merged, I will open a new one to address it, so this should not stay there for long

Reply:
Added comments, LMK what you think